requirements-dev.txt

```
@@ -9,7 +9,7 @@ numpy
 pillow
 pre-commit
 pluggy >= 1.4.0
-pytest > 7, < 8 # Datadog's ddtrace does not support pytest 8 yet. See https://github.com/DataDog/dd-trace-py/issues/8220
+pytest > 7, < 9 # Datadog's ddtrace does not support pytest 8 yet. See https://github.com/DataDog/dd-trace-py/issues/8220
```

```suggestion
pytest > 7, < 9
```
we don't depend on `ddtrace` anymore